src/biokbase/narrative/tests/test_job.py

```diff
@@ -147,33 +154,39 @@ def state_2_kwargs(state, transform_list=JOB_ATTRS):
     return kwargs
 
 
-def create_job_from_ee2(job_id):
+def create_job_from_ee2(job_id, mode, children=None):
```

Comment:
So the tests kept failing because they were old they would create a job from the attributes and the expected state from the whole ee2 state; but you "lose" information when you create a job from attributes. This may need more thought

Reply:
Yeah -- is it only the app manager that creates jobs from attributes? I think it might be worth ditching that special case and always having the job created from the ee2 state object -- at least then we KNOW that we have all the data available.

Reply:
And I was thinking, you can introduce weird bugs if you let yourself create a `Job` from very partial attributes and then you try to do a `Job.something()`

Reply:
Ticket to convert over to using the ee2 job state object: https://kbase-jira.atlassian.net/browse/DATAUP-556

That should also allow us to simplify both the code and the tests. 😁

Reply:
One option here would be for the AppManager to either pass the new Job Id over to the JobManager and have it check state and make a new job from it.  Or the AppManager could directly call `EE2.check_jobs` right after job creation to get the state object. I think either way would be fine.

Reply:
But I agree that having One True way to make Job objects is best and would avoid confusion and odd side cases. You're already removing the mixed state + attributes setup, so that's a good first step. 👍